types/errors/errors.go

```diff
@@ -37,135 +37,135 @@ const RootCodespace = "sdk"
 
```

Comment:
Shouldn't we remove the aliases from above?
They've been deprecated since v0.46.

Reply:
There was a build failing because x/upgrade was using this one iirc. Should we keep them until we tag new versions of everything and then remove it?

Reply:
Yeah makes sense